react-search-component/my-app/src/SearchableList.jsx

```diff
@@ -0,0 +1,18 @@
+import SearchBar from './SearchBar';
+import SearchResult from './SearchResult';
+import { useState } from 'react';
+
+export default function SearchableList({ items }) {
+  const [search, setSearch] = useState('');
+
+  function searchInput(text) {
+    setSearch(text);
+  }
```

Comment:
This is redundant! We can just pass the `setSearch` function instead in the prop!